crab-build.file

```diff
@@ -40,7 +40,7 @@ echo dbs >> %{i}/etc/crab_py_pkgs.txt
 echo RestClient >> %{i}/etc/crab_py_pkgs.txt
 
 #Copy WMCore
-if [ "%{crab_type}" != "dev" ] ; then
+if [ "%{crab_type}" == "pre" ] ; then
```

Comment:
@ddaina , is this what you want? `pre` will use `WMCore` and `prod` and `dev` will use   `CRABClient/WMCoreConfiguration.py` ?

Reply:
yes, that's what we want. We use `pre` as `previous` so users can fall back to it in case of problems when we move from `dev` to `prod`. We do not have means to do a menaingful validation in "preprod" after we do our best in `dev` which in this case includes having CMSSW_12 users use it !

Reply:
my bad, I thought `pre` stands for for pre-production/pre-release :-)

Reply:
no reason to feel bad. It was a dirty expedient I dreamed off. And not standard in any way. Yes, it is a devious use of the `pre` prefix , sorry about that. Did not want to ask you to introduce a `crab-old`